src/mlpack/methods/kmeans/kmeans_main.cpp

```diff
@@ -91,21 +91,7 @@ PROGRAM_INFO("K-Means Clustering",
     "following command may be used:"
     "\n\n" +
     PRINT_CALL("kmeans", "input", "data", "initial_centroids", "initial",
-        "clusters", 10, "max_iterations", 500, "centroid", "final"),
-    SEE_ALSO("K-Means tutorial", "@doxygen/kmtutorial.html"),
```

Comment:
Hm, this changes are still missing, did you rebased the PR against the latest master branch?

Reply:
i did rebased the pr against the latest master branch.Playing around with the code i may have replaced the latest version .